yejin/Algorithm/5월/PGS/[1차] 셔틀버스/[1차] 셔틀버스.java

```diff
@@ -0,0 +1,89 @@
+import java.util.*;
+import java.io.*;
+
+
+class Solution {
+    public String solution(int n, int t, int m, String[] timetable) {
+        String answer = "";
+        // 셔틀 운행 횟수 n, 셔틀 운행 간격 t, 한 셔틀에 탈 수 있는 최대 크루 수 m, 크루가 대기열에 도착하는 시각을 모은 배열 timetable
+        
+        PriorityQueue<Integer> pq = new PriorityQueue<>();
```

Comment:
와~! qp를 쓰면 정렬을 따로 하지 않아 줘도 되서 너무 편하겠어요~!

Reply:
If you look at someone else's solution, there is a solution that was solved using `sort`, but I wonder why you used priority queue instead of sorting.

Reply:
다른 사람의 솔루션을 보면 정렬을 사용하여 해결한 솔루션이 있는데 왜 정렬 대신 우선순위 큐를 사용했는지 궁금합니다.